chain/client/src/sync/epoch.rs

```diff
@@ -487,8 +488,15 @@ impl Handler<EpochSyncRequestMessage> for ClientActorInner {
                         return;
                     }
                 };
+                let (compressed_proof, _) = match CompressedEpochSyncProof::encode(&proof) {
+                    Ok(compressed_proof) => compressed_proof,
+                    Err(e) => {
+                        tracing::error!("Failed to compress epoch sync proof: {:?}", e);
```

Comment:
nit: best to use structured logging e.g.:
Err(err) => { tracing::error!(?err, "Failed to .."); ...